TYPO3.Neos/Configuration/Settings.yaml

```diff
@@ -161,6 +161,33 @@ TYPO3:
         'ru': 'Pусский – Russian'
 #        'sv': 'Svenska – Swedish'
 
+      # Aloha presets used in nodeType configuration
+      aloha:
+        presets:
+          'Standard':
```

Comment:
More configs out of the box make sense indeed. I usually call them PlainText (disabling everything) and RichText (bold + italic + h2-h3 + links) configs. Maybe have three presets `plain`, `rich` and `full`?